devenv.yaml

```diff
@@ -0,0 +1,14 @@
+inputs:
+  nixpkgs:
+    url: github:cachix/devenv-nixpkgs/rolling
```

Comment:
What is this URL for?

Reply:
its simply configured that way by running `devenv init`